src/integration-tests/src/secret_manager/openapi.rs

```diff
@@ -23,9 +23,7 @@ pub async fn run() -> Result<()> {
         .map(char::from)
         .collect();
 
-    let client = smo::Client::new()
-        .await?
-        .google_cloud_secretmanager_v_1_secret_manager_service();
+    let client = smo::GoogleCloudSecretmanagerV1SecretManagerServiceClient::new().await?;
```

Comment:
No change needed in this PR, but I think #235 is really relevant.

Reply:
Yeah, I think we need a codec option for this if we can't programmatically come up with a good name for openAPI clients. We have this in a couple languages today as a part of the per-language service config.